saexec/subscription.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package saexec
+
+import (
+	"github.com/ava-labs/libevm/core"
+	"github.com/ava-labs/libevm/core/types"
+	"github.com/ava-labs/libevm/event"
+)
+
+func (e *Executor) sendPostExecutionEvents(b *types.Block, receipts types.Receipts) {
+	e.headEvents.Send(core.ChainHeadEvent{Block: b})
```

Comment:
Just commenting that in the geth code they don't always public a chain head event (due to performance concerns). I'm not sure that is relevant for us... But I figured I'd point it out as a weird quirk that they do.